### ISSUE TITLE
Does not work autocomplete after generation @InjectView

### ISSUE THREAD

**Issue by IlyaEremin** [avast/android-butterknife-zelezny#52]
Android Studio 1.3 preview 3.0. I faced with this issue on the earlier versions of AS. It is only for windows (I use windows 8.1). MacOS works correct.

**Comment by prashantwosti** [comment]
Same here. I'm on Windows 10. Android Studio 1.4.1.

**Comment by destil** [comment]
What do you mean 'autocomplete doesn't work'? Autocomplete in whole AS?

**Comment by prashantwosti** [comment]
yes. when you generate injections, auto code completion does not work in your whole project for a while. 
I have experienced this so many times.

**Comment by destil** [comment]
I have never experienced this. Do you have some sample project I can use to reproduce the issue?

**Comment by prashantwosti** [comment]
@destil This issue is not project specific. I encounter it always. 
Below is my log i recently encountered in Android studio 1.5.1 | Windows10

```
java.lang.NullPointerException
	at com.avast.android.butterknifezelezny.InjectAction.onConfirm(InjectAction.java:95)
	at com.avast.android.butterknifezelezny.form.EntryList$ConfirmAction.actionPerformed(EntryList.java:213)
	at javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:2022)
	at javax.swing.AbstractButton$Handler.actionPerformed(AbstractButton.java:2346)
	at javax.swing.DefaultButtonModel.fireActionPerformed(DefaultButtonModel.java:402)
	at javax.swing.DefaultButtonModel.setPressed(DefaultButtonModel.java:259)
	at javax.swing.plaf.basic.BasicButtonListener.mouseReleased(BasicButtonListener.java:252)
	at java.awt.Component.processMouseEvent(Component.java:6525)
	at javax.swing.JComponent.processMouseEvent(JComponent.java:3324)
	at java.awt.Component.processEvent(Component.java:6290)
	at java.awt.Container.processEvent(Container.java:2234)
	at java.awt.Component.dispatchEventImpl(Component.java:4881)
	at java.awt.Container.dispatchEventImpl(Container.java:2292)
	at java.awt.Component.dispatchEvent(Component.java:4703)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4898)
	at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4533)
	at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4462)
	at java.awt.Container.dispatchEventImpl(Container.java:2278)
	at java.awt.Window.dispatchEventImpl(Window.java:2750)
	at java.awt.Component.dispatchEvent(Component.java:4703)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:758)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:75)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.awt.EventQueue$4.run(EventQueue.java:731)
	at java.awt.EventQueue$4.run(EventQueue.java:729)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:75)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:728)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:734)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:565)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:382)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```
Hope you get it now.

**Comment by destil** [comment]
Fix for this crash was just released in v1.4.0
